src/main/java/seedu/address/model/transaction/TransactionLog.java

```diff
@@ -38,6 +38,9 @@ public List<Transaction> getTransactionLog() {
      */
     public double calculateNetTransacted() {
         double result = 0;
+        if (transactionList.isEmpty()) {
+            return result;
+        }
```

Comment:
I don't think this is needed. If transactionList is empty, the for loop below would exit immediately and go to
```java
return result;
```
which will return 0.

Reply:
You're right. Good catch! I've made amendments to the code.